src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte

```diff
@@ -0,0 +1,471 @@
+              colspan={pinnedColumns.length || 1}
+            >
+              {#if allSelected}
+                <span class="font-semibold">
+                  All {filteredWorkflowCount} selected
+                </span>
+              {:else}
+                <span class="font-semibold"
+                  >{selectedWorkflowsCount} selected</span
+                >
+                <span>
+                  (or <button
+                    data-testid="select-all-workflows"
+                    on:click={handleSelectAll}
+                    class="cursor-pointer underline"
+                    >select all {filteredWorkflowCount}</button
+                  >)
+                </span>
+              {/if}
+              <div class="ml-4 inline-flex gap-2">
+                {#if cancelEnabled}
+                  <BulkActionButton
+                    testId="bulk-cancel-button"
+                    disabled={namespaceWriteDisabled}
+                    on:click={() => dispatch('cancelWorkflows')}
+                    >Request Cancellation</BulkActionButton
+                  >
+                {/if}
+                {#if terminateEnabled}
+                  <BulkActionButton
+                    variant="destructive"
+                    testId="bulk-terminate-button"
+                    disabled={namespaceWriteDisabled}
+                    on:click={() => dispatch('terminateWorkflows')}
+                    >Terminate</BulkActionButton
+                  >
+                {/if}
+              </div>
+            </th>
+          {:else}
+            {#each pinnedColumns as column}
+              <WorkflowsSummaryTableHeaderCell {column} {sortDisabled} />
+            {/each}
+          {/if}
+        </tr>
+        {#if $updating}
+          <ProgressBar />
+        {/if}
+      </thead>
+      <tbody>
+        {#each workflows as workflow}
+          <tr
+            class="workflow-summary-row pinned"
+            on:click={() => goToWorkflow(workflow)}
+          >
+            {#if $supportsBulkActions}
+              <td>
+                <Checkbox
+                  hoverable
+                  bind:group={selectedWorkflows}
+                  value={workflow}
+                  disabled={allSelected}
+                />
+              </td>
+            {/if}
+            {#each pinnedColumns as column}
+              <WorkflowsSummaryTableBodyCell {column} {workflow} />
+            {/each}
+          </tr>
+        {/each}
+      </tbody>
+    </table>
+  </div>
+  <div class="workflow-summary-table-wrapper">
+    <table class="workflow-summary-table">
+      <thead>
+        <tr class="workflow-summary-header-row">
+          {#each otherColumns as column}
+            <WorkflowsSummaryTableHeaderCell {column} {sortDisabled} />
+          {/each}
+          <th
+            class="px-2 h-10 w-12 text-right sticky right-0 flex items-center justify-center bg-primary"
+          >
+            <IconButton
```

Comment:
Might we want to make the ellipsis button always hug the right? It floats to the left if there aren't enough headers.

![Screenshot 2023-03-27 at 3 49 25 PM](https://user-images.githubusercontent.com/15069288/228077460-5e7c8fea-44ba-4731-bf4d-b3b1f0e13acc.png)
![Screenshot 2023-03-27 at 3 57 26 PM](https://user-images.githubusercontent.com/15069288/228077474-d1394571-9082-4967-b051-ba9fd5cd1a1e.png)

Reply:
Yes, good catch, should be fixed: [296e2d6](https://github.com/temporalio/ui/pull/1229/commits/296e2d663ea227a8f2918439e86c6deb0b06a9e3)